base/uk.ac.stfc.isis.ibex.ui.configserver/src/uk/ac/stfc/isis/ibex/ui/configserver/ImportConverter.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * This file is part of the ISIS IBEX application.
+ * Copyright (C) 2012-2023 Science & Technology Facilities Council.
+ * All rights reserved.
+ *
+ * This program is distributed in the hope that it will be useful.
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v1.0 which accompanies this distribution.
+ * EXCEPT AS EXPRESSLY SET FORTH IN THE ECLIPSE PUBLIC LICENSE V1.0, THE PROGRAM 
+ * AND ACCOMPANYING MATERIALS ARE PROVIDED ON AN "AS IS" BASIS, WITHOUT WARRANTIES 
+ * OR CONDITIONS OF ANY KIND.  See the Eclipse Public License v1.0 for more details.
+ *
+ * You should have received a copy of the Eclipse Public License v1.0
+ * along with this program; if not, you can obtain a copy from
+ * https://www.eclipse.org/org/documents/epl-v10.php or 
+ * http://opensource.org/licenses/eclipse-1.0.php
+ */
+
+package uk.ac.stfc.isis.ibex.ui.configserver;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.stream.Collectors;
+
+import uk.ac.stfc.isis.ibex.configserver.configuration.Configuration;
+import uk.ac.stfc.isis.ibex.configserver.configuration.PVDefaultValue;
+import uk.ac.stfc.isis.ibex.configserver.configuration.PVSet;
+import uk.ac.stfc.isis.ibex.configserver.editing.DuplicateBlockNameException;
+import uk.ac.stfc.isis.ibex.configserver.editing.EditableBlock;
+import uk.ac.stfc.isis.ibex.configserver.editing.EditableConfiguration;
+import uk.ac.stfc.isis.ibex.configserver.editing.EditableIoc;
+import uk.ac.stfc.isis.ibex.configserver.internal.DisplayUtils;
+
+/**
+ * Handles the changes to be done when importing a component from another instrument.
+ */
+public final class ImportConverter {
```

Comment:
Doesn't this class belong in the models (`uk.ac.stfc.isis.ibex.configserver`) rather than the views (`uk.ac.stfc.isis.ibex.ui.configserver`)? Can you check all similar logic?

Views and viewmodels belong in `.ui`, back-end classes don't.